contracts/Core/StakeManager.sol

```diff
@@ -100,7 +100,6 @@ contract StakeManager is Initializable, StakeStorage, StateManager, Pause, Stake
         uint256 totalSupply = 0;
 
         if (stakerId == 0) {
-            require(amount >= minStake, "Amount below Minstake");
```

Comment:
wouldnt this allow 0 razor also? maybe we should alloe minimum 1e18 razor? or more according to governance?

Reply:
yep I had same thought, but it doesn't create any issue on code
0 sRZRs will be minted

Reply:
Please add tests to validate this @shekhar2807

Reply:
Added test for this, Staker is able to stake zero razor. @hrishikeshio @SkandaBhat

Reply:
Move minimum `1e18 RAZOR` requirement to governance parameter and check here.

Reply:
Done @SkandaBhat